frontend-thea-aa/src/lib/types.ts

```diff
@@ -1,23 +1,37 @@
 export type LatLong = {
-    lat: number
-    long: number
+	lat: number
```

Comment:
you appear to have different tab size on your ide, I don't really care but worth noting